wrapper/src/test/java/software/amazon/jdbc/plugin/cache/CachedResultSetTest.java

```diff
@@ -368,10 +370,15 @@ void test_get_special_time() throws SQLException {
         4362000L,
         LocalTime.of(10, 20, 30),
         OffsetTime.of(12, 15, 30, 0, ZoneOffset.UTC),
+        new Timestamp(1755621000000L), // Date and time (GMT): Tuesday, August 19, 2025 4:30:00 PM
+        new Timestamp(1735713000000L), // Date and time (GMT): Wednesday, January 1, 2025 6:30:00 AM
+        new Timestamp(0L), // 1970-01-01 00:00:00 UTC (epoch)
+        new Timestamp(1787416200000L), // Future Date: Date and time (GMT): Saturday, August 22, 2026 4:30:00 PM
```

Comment:
I meant you make this one year from `now`. 

If you hard-code it to 2026 August, this test will no longer be testing a future date once we get to Aug 2026 :)